.github/ISSUE_TEMPLATE/2_feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: 'Feature request'
+about: 'Suggest a feature for miranum'
+title:
+labels: 'type: enhancement'
+assignees: dominikhorn93
+---
+
+### Scenario
+
+* miranum-connect version:
+* miranum-platform version:
+* Camunda version:
+* Description of your use case: (detailed description or executable reproducer, e.g. GitHub repo)
+
+### Current Behaviour
+
+### Wanted Behaviour
+
+### Possible Workarounds
```

Comment:
```suggestion
### Is your feature request related to a problem? Please describe
<!-- A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]-->
<!-- Is there a time frame or deadline for the topic? -->

### Describe the solution you'd like
<!-- A clear and concise description of what you want to happen.-->

### Describe alternatives you've considered
<!-- A clear and concise description of any alternative solutions or features you've considered.-->

### Additional context
<!-- Add any other context or screenshots about the feature request here.-->

```

Reply:
In case of a feature request I would like to suggest a new feature. Usually the current behaviour is *feature is not there** and the wanted behaviour is the feature.
The versions of miranum should be the latest version for a feature request. I don't think we want to support old versions of miranum.